recipes/galeodes/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "galeodes" %}
+{% set version = "0.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/galeodes-{{ version }}.tar.gz
+  sha256: 92465cb8485165c2889f90d708617b1f505ba484859836cf6e19f9545dcbe97d
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.5
+  run:
+    - pillow
+    - python >=3.5
+    - requests
+    - selenium
+
+test:
+  imports:
+    - galeodes
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/qeeqbox/galeodes
+  summary: Browsers options
+  license: AGPL-3.0-or-later
```

Comment:
Are you sure that it's `or-later`?

Reply:
https://github.com/qeeqbox/galeodes/blob/main/LICENSE#L635
```
    This program is free software: you can redistribute it and/or modify
    it under the terms of the GNU Affero General Public License as published
    by the Free Software Foundation, either version 3 of the License, or
    (at your option) any later version.
```